README.md

```diff
@@ -1,56 +1,42 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Tour Booking App (backend)
```

Comment:
- [ ] A screenshot of your readme would be awesome.  It could be the models diagram, or even your API docs:

![image](https://user-images.githubusercontent.com/57421823/199580253-cd2a1211-3e68-4b16-9563-0fddbfdfad28.png)

That way, your repo documentation will look more professional. 💪